packages/create-bison-app/template/README.md.ejs

```diff
@@ -60,97 +61,212 @@ Bison uses [Nexus Schema](https://nexusjs.org/docs/) to create the GraphQL API.
+    name: 'UpdateOrganizationInput',
+    description: 'Input used to update a organization',
+    definition: (t) => {
+      t.nonNull.string('name');
+    },
+  });
+
+  // Query Inputs
+  export const OrganizationOrderByInput = inputObjectType({
+    name: 'OrganizationOrderByInput',
+    description: 'Order organization by a specific field',
+    definition(t) {
+      t.field('name', { type: 'SortOrder' });
+    },
+  });
+
+  export const OrganizationWhereUniqueInput = inputObjectType({
+    name: 'OrganizationWhereUniqueInput',
+    description: 'Input to find organizations based on unique fields',
+    definition(t) {
+      t.id('id');
+      // add DB uniq fields here
+      // t.string('name');
+    },
+  });
+
+  export const OrganizationWhereInput = inputObjectType({
+    name: 'OrganizationWhereInput',
+    description: 'Input to find organizations based on other fields',
+    definition(t) {
+      t.field('name', { type: 'StringFilter' });
+    },
+  });
+  ```
+</details>
 
 ### Understanding the GraphQL API and TypeScript types
 - [ ] Open `api.graphql` and look at our the new definitions that were generated for you:
 
 ```graphql
+
 """
-An Organization
+A Organization
 """
 type Organization {
   createdAt: DateTime!
-  id: String!
+  id: ID!
+  name: String!
   updatedAt: DateTime!
 }
 
-input OrganizationCreateOneWithoutUsersInput {
-  connect: OrganizationWhereUniqueInput
-  create: OrganizationCreateWithoutUsersInput
-}
-
-input OrganizationCreateWithoutUsersInput {
-  createdAt: DateTime
-  id: String
-  updatedAt: DateTime
+"""
+Order organization by a specific field
+"""
+input OrganizationOrderByInput {
+  name: SortOrder
 }
 
+"""
+Input to find organizations based on other fields
+"""
 input OrganizationWhereInput {
-  AND: [OrganizationWhereInput!]
-  createdAt: DateTimeFilter
-  id: StringFilter
-  NOT: [OrganizationWhereInput!]
-  OR: [OrganizationWhereInput!]
-  updatedAt: DateTimeFilter
-  users: UserListRelationFilter
+  name: StringFilter
 }
 
+"""
+Input to find organizations based on unique fields
+"""
 input OrganizationWhereUniqueInput {
-  id: String
+  id: ID
+}
+
+"""
+Input used to create a organization
+"""
+input CreateOrganizationInput {
+  name: String!
+}
+
+"""
+Input used to update a organization
+"""
+input UpdateOrganizationInput {
+  name: String!
 }
 ```
 
-- [ ] Open up `types.ts` to see the generated TypeScript types that correspond with the graphql changes.
+- [ ] Open up `types.ts` to see the generated TypeScript types that correspond with the GraphQL changes.
 
 ### API Request Tests
 Let's confirm the API changes using a request test. To do this:
 - [ ] Generate a new factory: `yarn g:test:factory organization`
-- [ ] Add a default value for organization name in the build function. You can use any of the methods from the `chance` library.
+- [ ] Add a default value for organization name in the `build` function. You can use any of the methods from the `chance` library.
```

Comment:
Should we link to the chance library docs for easy reference?

Reply:
That's a good idea. I've added the link.